daemon/mounts.go

```diff
@@ -8,13 +8,21 @@ import (
 	mounttypes "github.com/docker/docker/api/types/mount"
 	"github.com/docker/docker/container"
 	volumesservice "github.com/docker/docker/volume/service"
+	"github.com/sirupsen/logrus"
 )
 
 func (daemon *Daemon) prepareMountPoints(container *container.Container) error {
+	alive := container.IsRunning()
 	for _, config := range container.MountPoints {
 		if err := daemon.lazyInitializeVolume(container.ID, config); err != nil {
 			return err
 		}
+		if alive {
+			logrus.WithField("container", container.ID).WithField("volume", config.Volume.Name()).Debug("Live-restoring volume for alive container")
```

Comment:
We probably want to adjust the logging to the new containerd logger first (https://github.com/moby/moby/pull/45799)

Reply:
Yeah, I saw this one; I'm okay with fixing up after this though

Reply:
^^ I should add that one reason why doing it later is that we're backporting this fix to other branches (which don't have the new logging) 😅

Reply:
Ah, that makes sense!

Reply:
That being said, I'd like this to be a `WithFields(logrus.Fields{})` for all branches before merge, otherwise this looks good.